foundation-ui/utils/ReactUtil.js

```diff
@@ -15,6 +15,10 @@ const ReactUil = {
       return null;
     }
 
+    if (Array.isArray(elements) && elements.length === 1 && !alwaysWrap) {
```

Comment:
Yeah, good point...  We would need to check all elements and that's probably a little over the top and definitely not blocking this PR.